specifyweb/frontend/js_src/lib/components/Attachments/RecordSetAttachment.tsx

```diff
@@ -81,6 +85,36 @@ export function RecordSetAttachments<SCHEMA extends AnySchema>({
   );
   const attachmentsRef = React.useRef(attachments);
 
+  const handleDownloadAllAttachments = async (): Promise<void> => {
+    if (attachmentsRef.current === undefined) return;
+    const attachmentLocations: readonly string[] = attachmentsRef.current.attachments
```

Comment:
don't need explicit type annotation - typescript can infer it

```suggestion
    const attachmentLocations = attachmentsRef.current.attachments
```